src/kraken/std/util/exists_and_committed.py

```diff
@@ -3,10 +3,10 @@
 import subprocess as sp
 from pathlib import Path
 
-from kraken.core.api import Property, Task, TaskStatus
+from kraken.core.api import Project, Property, Task, TaskStatus
 
 
-class ImportantFileCheckTask(Task):
+class ExistsAndCommittedTask(Task):
```

Comment:
nit: Make the purpose of the task more obvious

```suggestion
class CheckFileExistsAndIsCommittedTask(Task):
```